MIDAS/src/gnc/example_kf.h

```diff
@@ -0,0 +1,29 @@
+#ifndef GNC_EXAMPLE_KF_H_
```

Comment:
Any reason we're doing include guards instead of pragmas?

Reply:
if magilan gets to die on a hill I'm gonna die on the hill of using guards instead of pragma, they said not to use pragma at blue and here at relativity

Reply:
yeah but why not use pragmas

Reply:
harder to break the codebase with pragmas is the only reason i push a bit

Reply:
Given how simple our codebase is, I don't see any reason to not use #pragma once. It's just more concise, readable, and less error-prone.

Reply:
fine i wont die on the hill